decidim-type/spec/decidim/organizations_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "decidim/api/test/type_context"
+
+describe "Organizations", type: :system do
+  let(:organization) { create(:organization) }
+  let(:locale) { "en" }
+  let(:user) { create(:user, organization: organization, admin: true, nickname: "admin") }
+
+  it "has created a relationship with user admin" do
+    expect(organization.admins.or(organization.users_with_any_role)).to eq([user])
+  end
+
+  let(:organization_process_admin) { create(:organization) }
+  let(:user_process_admin) { create(:user, organization: organization_process_admin, admin: false, nickname: "admin_process") }
+  let(:participatory_process) { create(:participatory_process, organization: organization_process_admin, slug: 'processtest')}
+  let(:participatory_process_user_role) { create(:participatory_process_user_role, user: user_process_admin, participatory_process: participatory_process)}
+
+  it "has created a relationship with user process admin" do
+    expect([participatory_process_user_role.user]).to eq(organization_process_admin.admins.or(organization_process_admin.users_with_any_role))
+  end
+
+  let(:organization_assembly_admin) { create(:organization) }
```

Comment:
why do we have 3 organizations?
I think we can reuse the same `organization` for all tests. Tests are atomic, on each test (each `it` statement) the DDBB is created and rolled back.